lib/zipkin-tracer/rack/zipkin-tracer.rb

```diff
@@ -50,8 +50,8 @@ def annotate_plugin(env, status, response_headers, response_body)
     def trace!(span, zipkin_env, &block)
       # if the request comes from a non zipkin-enabled source record the default tags
       tags = DEFAULT_SERVER_RECV_TAGS unless zipkin_env.called_with_zipkin_headers?
-      # if the user specified tags to record on server recv, use these no matter what
-      tags = @config.record_on_server_receive if @config.record_on_server_receive
+      # if the user specified tags to record on server receive, use these no matter what
+      tags = @config.record_on_server_receive unless @config.record_on_server_receive.empty?
```

Comment:
ah, nope, can't do this because of the case where we'd been called by a non zipkin enabled service, `tags` would be using the `DEFAULT_SERVER_RECV_TAGS`. Getting rid of the check here would overwrite this and make it empty...
And I can't use that `DEFAULT_SERVER_RECV_TAGS` as the config's default altogether because we still need it to be empty in the 99% case where we're called with zipkin headers and we don't want the http.path to be recorded too.
In a nutshell, we need the empty default config while having an alternative default for the case of no zipkin headers in the request.
Makes sense?